### ISSUE TITLE
Failed to connect to the remote extension host server (Error: Failed to connect to relay.)

### ISSUE THREAD

**Issue by fpintos** [MicrosoftDocs/vsonline#336]
My remote self-host machine rebooted to install a new Windows Insider build and now I'm unable to connect to its environment in vsonline, even thou it reports as being available in the environments web page.

The client machine reports:
Failed to connect to the remote extension host server (Error: Failed to connect to relay.)
 
I've unblocked myself by registering the same folder as a different environment, and now I can connect to this 2nd environment, but still, the 1st one fails with the relay error.

NOTE: As I was about to post this, GitHub suggested #5 , and in there it mentions 'Restore Local Environment', and this indeed worked. So I suppose the request at this point is for local environments to be restored automatically on reboot.

Thanks

**Comment by lostintangent** [comment]
Thanks for reporting this! As you found, we’re planning to address this issue via #5, and we’re actually very close to shipping this support. I’m going to close this issue as a duplicate of #5, so stay tuned to that issue for future updates soon 👍
